src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -125,6 +125,14 @@
      :style {:margin-top 15}}
     (i18n/label :t/no-routes-found)]])
 
+(defn- not-enough-asset
+  []
+  [quo/alert-banner
+   {:action?         true
+    :text            (i18n/label :t/not-enough-assets)
+    :button-text     (i18n/label :t/buy-eth)
+    :on-button-press #(js/alert "not implemented yet")}])
```

Comment:
for now we should be feature flagging incomplete functionality instead of showing to an end user 👍

Reply:
but I think we have the buy button functionality already 🤔 
cc @ajayesivan

Reply:
Right, I forgot about it. Thanks for letting me know

Reply:
Also to facilitate later on due to searchability, it's a bit better to reuse `status-im.common.not-implemented/alert` if we can.

> for now we should be feature flagging incomplete functionality instead of showing to an end user 

@J-Son89, we have `config/show-not-implemented-features?`. Is that what you mean? I see we could have this same function implemented using the new feature flag system. I almost missed it exists in `config`.

Reply:
We have this: Should I show it here?

<img width="397" alt="Screenshot 2024-06-10 at 13 37 16" src="https://github.com/status-im/status-mobile/assets/55688834/2ee27c65-132a-4bfa-9ad8-dd8ac37d1b7e">

Reply:
> We have this: Should I show it here?
> 
> <img alt="Screenshot 2024-06-10 at 13 37 16" width="397" src="https://private-user-images.githubusercontent.com/55688834/338168539-2ee27c65-132a-4bfa-9ad8-dd8ac37d1b7e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgwMjE5NDgsIm5iZiI6MTcxODAyMTY0OCwicGF0aCI6Ii81NTY4ODgzNC8zMzgxNjg1MzktMmVlMjdjNjUtMTMyYS00YmZhLTlhZDgtZGQ4YWMzN2QxYjdlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjEwVDEyMTQwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg4NmZlMTk3Njg1NTNmYTI5ZDg3ZTAxM2I4ZDc0YjRjYWFjZWEyNDM2ZjFkMmVkM2UzYWEzNzYwYzA2MDhlZWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.D-3g6zj9-iPTGz4HrIg3OWBL-f5fMBMxd7C0GlTxjlQ">

looks like it: https://www.figma.com/design/xLs1KYmF4e6WwRTZVJKeUK/Descoped-Wallet?node-id=15329-133161&t=gkOnG71oLOAwwtNe-4

Reply:
thanks @mmilad75! 🙌